libs/core/src/stories/layout.docs.mdx

```diff
@@ -0,0 +1,420 @@
+
+import { Meta, Canvas, ArgTypes } from '@storybook/blocks';
+
+<Meta title="Foundations/Layout"/>
+
+# Layout
+The Layout system consists on using the `Row` and the `Box` component in tandem
+
+## Row
+The `pds-layout-row` component is designed to create horizontal rows that contain a set of columns.
+
+## Box
+The `pds-layout-box` component serves as a versatile building block for creating individual elements within your layout. You can use the `pds-layout-box` component to create columns, rows, and other layout elements with use of the `size` and `offset` properties or their variants.
+
```

Comment:
Should we maybe link to the components here as well? I know they're at the bottom so def not a show-stopper.